lib/runner/process-listener.js

```diff
@@ -40,8 +41,8 @@ module.exports = class {
     this.exit();
   }
 
-  unhandled(err) {
-    Logger.warn(`unhandledRejection: ${err.message}`);
+  unhandled(err, promise) {
+    Logger.error(`unhandledRejection: '${err}' at: ${util.inspect(promise)}`);
```

Comment:
Is `promise` passed in node v10?

Reply:
Yes, it is available for latest 10.x version: https://nodejs.org/docs/latest-v10.x/api/process.html#process_event_unhandledrejection